src/configure/configure.ts

```diff
@@ -122,11 +120,11 @@ class PipelineConfigurer {
             this.inputs.githubPATToken = await this.getGitHubPATToken();
         }
 
-        await this.getAzureDevOpsDetails();
-
         if (!this.inputs.targetResource.resource) {
             await this.getAzureResourceDetails();
         }
+
+        await this.getAzureDevOpsDetails();
```

Comment:
will this not change the order in which inputs are asked from user ?

Reply:
Yes, but this is inevitable. We cannot get right credential to make REST calls until we let user choose a subscription.

Reply:
Okay, this makes sense.
Just for my knowledge, do we get multiple sessions on single login

Reply:
Got it from the description